CONTRIBUTING.md

```diff
@@ -44,6 +44,33 @@ make the code easier to navigate.
 
 *(TODO: Provide some notes on how to prepare a development environment which
 includes the needed dependencies.)*
+### Note
+This project requires libwpewebkit-1.0-dev, libwpe-1.0-dev, libwpebackend-fdo-1.0-dev 
```

Comment:
It would be good to link to the respective GitHub repositories of these projects - perhaps present this as a bullet-point list. Maybe this could be split into separate 'Dependencies' and 'Building' sections?